chart/openfaas/README.md

```diff
@@ -48,6 +48,22 @@ kubectl -n openfaas create secret generic basic-auth \
 --from-literal=basic-auth-password="$PASSWORD"
 ```
 
+Generate secrets so that we can enable non-repudiation for asynchronous function callbacks:
+
+```bash
+rm signing.key > /dev/null 2>&1 || true && rm signing.key.pub > /dev/null 2>&1 || true
+ssh-keygen -t rsa -b 2048 -N "" -m PEM -f signing.key > /dev/null 2>&1
+openssl rsa -in ./signing.key -pubout -outform PEM -out signing.key.pub > /dev/null 2>&1
```

Comment:
Does every PC have `openssl`? If not, can this be done in an alpine container? Does that even make sense?

Reply:
seeing as it's advanced shall we leave for now, when we do `faas install` we will re-write in golang in anycase. `linux` and `mac` come with openssl pre-installed, windows does not .